014/client/src/apis/fetchContentsAPI.js

```diff
@@ -0,0 +1,27 @@
+const API_ENDPOINT = 'http://localhost:3000/api/content';
```

Comment:
```suggestion
const API_ENDPOINT = '/api/content';
```
이렇게 사용할 수 있도록 개발 환경을 구성해보면 어떨까요?